submissions/drewzag/task_js_dom/index.js

```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const getData = async () =>
+    fetch('/src/data.json')
+      .then((res) => {
+        if (res.ok) return res.json()
+        else throw new Error('Something went wrong')
+      })
+      .catch((error) => {
+        console.log(error)
+      })
```

Comment:
[Read this article](https://www.tjvantoll.com/2015/09/13/fetch-and-errors/) about handling failed HTTP responses with fetch()